blog/en/blog/2025/03/31/march-monthly-report.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "March Report (March 01 - March 31)"
+keywords: ["Apache APISIX", "API Gateway", "Monthly Report", "Contributor"]
+description: Our monthly Apache APISIX community report generates insights into the project's monthly developments. The reports provide a pathway into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
+tags: [Community]
+image: https://static.api7.ai/uploads/2025/03/28/226yQ3dc_march-monthly-report-cover-en.webp
+---
+
+> We have recently added some new features within Apache APISIX, including adding supports for `ai-prompt-guard`, `ai-rate-limiting`, and `ai-request-rewrite` plugins, and some new/enhanced features like adding JWT Audience Validator in `openid-connect` Plugin. For detailed information, please read the monthly report.
+<!--truncate-->
+
+## Introduction
+
+From its inception, the Apache APISIX project has embraced the ethos of open-source community collaboration, propelling it into the ranks of the most active global open-source API gateway projects. The proverbial wisdom of 'teamwork makes the dream work' rings true in our way and is made possible by the collective effort of our community.
+
+From March 1st to March 31, 14 contributors made 50 commits to Apache APISIX. We sincerely appreciate your contributions to Apache APISIX.
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://static.api7.ai/uploads/2025/03/31/fDYTi9Vu_march-contributor-list.webp)
+
+![New Contributors List](https://static.api7.ai/uploads/2025/03/31/NB9qpu7Q_mar-new-contributors.webp)
+
+## Good First Issue
+
+### Issue #12098 Improve Configuration Code of `chaitin-waf` Plugin
+
+Link：[https://github.com/apache/apisix/issues/12098](https://github.com/apache/apisix/issues/12098)
+
+Issue Description: Consider the comments of [PR #12029](https://github.com/apache/apisix/pull/12029#discussion_r2018012041) for `chaitin-waf` plugin, the value "nil" of `enum` can be removed due to the default value is set to "nil".
```

Comment:
```suggestion
Consider the comments of [PR #12029](https://github.com/apache/apisix/pull/12029#discussion_r2018012041) for `chaitin-waf` plugin. The value `nil` of `enum` can be removed since default value is already set to `nil`.
```